controllers/dnspolicy_controller_multi_cluster_test.go

```diff
@@ -194,46 +203,46 @@ var _ = Describe("DNSPolicy Multi Cluster", func() {
 									"ManagedZoneRef": HaveField("Name", "mz-example-com"),
 									"Endpoints": ConsistOf(
 										PointTo(MatchFields(IgnoreExtras, Fields{
-											"DNSName":       Equal("2w705o.lb-" + lbHash + ".test.example.com"),
+											"DNSName":       Equal(cgwTarget.GetShortCode() + "." + lbHash + ".test.example.com"),
```

Comment:
If you use `cgwTarget.GetShortCode()`  in the assertion like this you are never really testing what you expect the DNSName to be. What if there is a bug in  `cgwTarget.GetShortCode()` and it's not generating what we expect?

Reply:
Actually, there is a bug and this change is hiding it, some of these tests should fail because of https://github.com/Kuadrant/kuadrant-operator/pull/493#discussion_r1537350894 

You can see some of the cluster hashes were different previously, now they will all be the same.

Reply:
hmm, so what do we expect the hostnames to look like in multicluster? `<plaintext clusterid>-<gateway-hash>...` is what I was getting, is that correct?

Reply:
ok, I think I understand now, reviewing the code again.